ios/RNAdMobManager/RNAdmobNativeAdsManager.m

```diff
@@ -67,12 +67,12 @@ @implementation RNAdmobNativeAdsManager
     
     if ([[config allKeys] containsObject:@"tagForChildDirectedTreatment"]) {
         NSNumber *tag = [config valueForKey:@"tagForChildDirectedTreatment"];
-        [[[GADMobileAds sharedInstance] requestConfiguration] tagForChildDirectedTreatment:tag.boolValue];
+        [[[GADMobileAds sharedInstance] requestConfiguration] setTagForChildDirectedTreatment:[NSNumber numberWithBool:tag.boolValue]];
     };
     
     if ([[config allKeys] containsObject:@"tagForUnderAgeConsent"]) {
         NSNumber *tagC = [config valueForKey:@"tagForUnderAgeConsent"];
-        [[[GADMobileAds sharedInstance] requestConfiguration] tagForUnderAgeOfConsent:tagC.boolValue];
+        [[[GADMobileAds sharedInstance] requestConfiguration] setTagForChildDirectedTreatment:[NSNumber numberWithBool:tagC.boolValue]];
```

Comment:
Is there not a tagForUnderAgeConsent alternative, as well? Or does both have to be setTagForChildDirectedTreatment?

Reply:
This did fix it though after running a build for iOS using Expo EAS.